Tests/XCDiffCoreTests/Library/StringExtensionsTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright 2019 Bloomberg Finance L.P.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+@testable import XCDiffCore
+import XCTest
+
+final class StringExtensionsTests: XCTestCase {
+    private var delimiter: String!
+
+    override func setUp() {
+        super.setUp()
+
+        delimiter = "::"
+    }
+
+    func test_split_empty_string() {
```

Comment:
We have very informal, but commonly used pattern for naming the tests:
`test<method_name>_[when]<condition>_[<expected_result>]`

* `when` is optional
* `expected result` is also optional 

i.e.
`testSplit_whenStringNotContainDelimiter`

Reply:
I like the style guide, renamed the test functions.